src/openrct2-ui/windows/Park.cpp

```diff
@@ -1065,7 +1065,20 @@ static void window_park_guests_paint(rct_window* w, rct_drawpixelinfo* dpi)
     // Graph
     screenPos = w->windowPos + ScreenCoordsXY{ widget->left + 47, widget->top + 26 };
 
-    Graph::Draw(dpi, gGuestsInParkHistory, 32, screenPos);
+    uint8_t cappedHistory[32];
+    for (size_t i = 0; i < std::size(cappedHistory); i++)
+    {
+        auto value = gGuestsInParkHistory[i];
+        if (value != std::numeric_limits<uint32_t>::max())
+        {
+            cappedHistory[i] = static_cast<uint8_t>(std::min<uint32_t>(value, 5000) / 20);
```

Comment:
I'm not sure I fully understand why 5000 was picked but this whole forth and back is a remap.
0-5000 is 0 to 250 and vice versa. 

So shouldn't 5000 be the maximum amount of guests possible?

Reply:
Not really, even in vanilla you can have more than 5000 guests (~9400 is the maximum). It’s just a nice round value that (divided by 20) just fitted in an uint8_t.

The graph drawing code should be modified to be able to take higher limits. I’ll create an issue for it.

Reply:
Well the graph will be off quite a bit it only does the first 5000, with NSF we can have a huge amount of peeps so this should be probably some sort of constant. 
We should also have a remap function that does remap(value, min1, max1, min1, max1), which would work for any range.

Reply:
https://github.com/OpenRCT2/OpenRCT2/issues/15350